src/WebJobs.Script.WebHost/Properties/Resources.resx

```diff
@@ -260,7 +260,7 @@
 }</value>
   </data>
   <data name="HostSpecializationTrace" xml:space="preserve">
-    <value>Host has been specialized</value>
+    <value>Starting host specialization</value>
```

Comment:
I think part of the confusion has been that this log statement seemed to indicate that specialization was done. However, we emit this log when we're _starting_ specialization